pages/common/rss2email.md

```diff
@@ -0,0 +1,28 @@
+# rss2email
+
+> Tool for delivering news from RSS feeds to your email program.
+> More information: <https://github.com/rss2email/rss2email>.
+
+- Open help page:
```

Comment:
Can you move this command to last?

Also update it to 
```suggestion
- Display help:
```
as this is what we use across all our pages.